setup.py

```diff
@@ -135,6 +133,17 @@ def _rewrite_ld_preload_cuda(self, to_preload):
                     f.write('    import os\n')
                     f.write('    os.environ["ORT_CUDA_UNAVAILABLE"] = "1"\n')
 
+        def _rewrite_ld_preload_tensorrt(self, to_preload):
+            with open('onnxruntime/capi/_ld_preload.py', 'a') as f:
+                if len(to_preload) > 0:
+                    f.write('from ctypes import CDLL, RTLD_GLOBAL\n')
+                    f.write('try:\n')
+                    for library in to_preload:
+                        f.write('    _{} = CDLL("{}", mode=RTLD_GLOBAL)\n'.format(library.split('.')[0], library))
+                    f.write('except OSError:\n')
+                    f.write('    import os\n')
+                    f.write('    os.environ["ORT_TENSORRT_UNAVAILABLE"] = "1"\n')
```

Comment:
pybind_state.cc needs to be modified to check if ORT_TENSORRT_UNAVAILABLE if set to 1? 
same as what's being done for ORT_CUDA_UNAVAILABLE

Reply:
yes, that's right. Will add in pybind_state.cc